custom_components/omnik_inverter/translations/en.json

```diff
@@ -19,6 +19,15 @@
                     "password": "Password"
                 }
             },
+            "setup_tcp": {
+                "title": "Omnik Inverter - TCP",
+                "description": "Set up Omnik Inverter to integrate with Home Assistant.",
+                "data": {
+                    "name": "Name",
+                    "host": "Host",
+                    "serial": "Serial Number"
```

Comment:
As it turns out the serial number may actually be optional in the request packet, the backend replies anyway. We can only (recommend the community to) test this if the assertion is removed from `python-omnikinverter` though.